### PR TITLE
Create helper for bumping dev version

### PR DESCRIPTION
**Pull request by CoderDake**
I've refactored `tool/update_version.dart` so that it's code can be shared with `tool/update_dev_version.dart`.

`tool/update_dev_version.dart` can now be run to bump the dev version.

## Testing
I've manually checked that:
- `-dev.0` is added if no dev version is present
- the `-dev.#` version is incremented when one is present.

### CODE REVIEW

File: tool/update_dev_version.dart [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import 'update_version.dart' as update_version;
```
Comment (bkonyi) [inline tool/update_dev_version.dart:1]: Add a copyright header to this file.

Comment (CoderDake) [reply tool/update_dev_version.dart:1]: 🤦

**Comment by elliette** [comment]
Let's update the documentation as well: https://github.com/flutter/devtools/blob/master/tool/README.md#update-the-devtools-version-number

Is there a reason to have this as a separate script instead of passing `dev` as an argument to `update_version.dart` (eg, `dart tool/update_version.dart dev` would update the dev version)?

**Comment by CoderDake** [comment]
@elliette no specific reason other than that is the way jacob brought it up.
The flag definitely sounds a bit cleaner, so I'll quickly do that.

**Review by elliette — APPROVED** [review]
This will be very helpful 🎉

File: tool/update_version.dart [hunk 2]
```diff
@@ -5,80 +5,90 @@
 import 'dart:convert';
 import 'dart:io';
 
+import 'package:args/command_runner.dart';
+
 // This script must be executed from the top level devtools/ directory.
 // TODO(kenz): If changes are made to this script, first consider refactoring to
 // use https://github.com/dart-lang/pubspec_parse.
```
Comment (CoderDake) [inline tool/update_version.dart:12]:
```suggestion
// use https://github.com/dart-lang/pubspec_parse.

```

**Pull request merged**